src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -115,23 +115,7 @@ protected override void ProcessRecord()
             {
                 if (ShouldProcess(path))
                 {
-                    try
-                    {
-                        AlternateDataStreamUtilities.DeleteFileStream(path, "Zone.Identifier");
-                    }
-                    catch (Win32Exception accessException)
-                    {
-                        // NativeErrorCode=2 - File not found.
-                        // If the block stream not found the 'path' was not blocked and we successfully return.
-                        if (accessException.NativeErrorCode != 2)
-                        {
-                            WriteError(new ErrorRecord(accessException, "RemoveItemUnauthorizedAccessError", ErrorCategory.PermissionDenied, path));
```

Comment:
This used to write out a non-terminating error, and now throw an exception which is a terminating error. That's not right.

Reply:
I agree. For reference https://docs.microsoft.com/en-us/powershell/developer/cmdlet/cmdlet-error-reporting#terminating-and-nonterminating-errors
>Is the error condition related to a specific input object or a subset of input objects? If so, this is a nonterminating error.